data_safe_haven/administration/users/guacamole_users.py

```diff
@@ -90,43 +55,8 @@ def list(self) -> Sequence[ResearchUser]:
 
     def remove(self, users: Sequence[ResearchUser]) -> None:
         """Remove list of users from Guacamole"""
-        if not users:
-            return
-        # Add user details to the mustache template
-        reader = FileReader(self.postgres_script_path / "remove_users.mustache.sql")
-        user_data = {
-            "users": [
-                {
-                    "username": user.user_principal_name,
-                }
-                for user in users
-            ]
-        }
-        for user in users:
-            self.info(
-                f"Removing '{user.preferred_username}' from SRE '{self.sre_name}' database"
-            )
-
-        # Create a temporary file with user details and run it on the Guacamole database
-        sql_file_name: Optional[pathlib.Path] = None
-        try:
-            with tempfile.NamedTemporaryFile("w", delete=False) as f_tmp:
-                f_tmp.writelines(reader.file_contents(user_data))
-                sql_file_name = pathlib.Path(f_tmp.name)
-            self.postgres_provisioner.execute_scripts([sql_file_name])
-            if self.users_:
-                self.users_ = [user for user in self.users_ if user not in users]
-        except Exception as exc:
-            raise DataSafeHavenInputException(
-                f"Could not update SRE '{self.sre_name}' database users.\n{str(exc)}"
-            ) from exc
-        finally:
-            if sql_file_name:
-                pathlib.Path(sql_file_name).unlink()
+        pass
 
     def set(self, users: Sequence[ResearchUser]) -> None:
         """Set Guacamole users to specified list"""
```

Comment:
```suggestion
```

Reply:
Sorry - I have implementations for these stubs but they aren't quite working atm.